lib/rubocop/cop/rails/helper_instance_variable.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for use of the helper methods which reference
+      # instance variables.
+      #
+      # Relying on the existing of a particular instance variable
```

Comment:
There are others reasons, but I'm still trying to find a good way to write out about.

Reply:
The wording is a bit off here. Consider:

> Relying on instance variables makes it difficult to re-use helper methods.

The next part about moving the behavior out of the helper sounds good to me. 👍

Reply:
Thanks, fixed in f78531e.